### PR TITLE
Updated documentation for User Settings API

### PR DESCRIPTION
**Pull request by mimrio**
**Signed-off.**

### CODE REVIEW

File: api-reference/beta/api/user_update_settings.md [hunk 1]
```diff
@@ -72,4 +73,14 @@ Content-length: 72
...
   "contributionToContentDiscoveryDisabled": true
 }
-```
\ No newline at end of file
+```
+
+##Batch request
+
+It's also possible to opt-out multiple users from Delve and disable their contribution on content relevancy for the whole organization through a batch request.
+Learn how to do it [here](https://developer.microsoft.com/en-us/graph/docs/concepts/json_batching).
```
Comment (lleonard-msft) [inline api-reference/beta/api/user_update_settings.md]: Typically, we use a more descriptive style to provide links, as in "To learn more, see [JSON batching](https://developer.microsoft.com/en-us/graph/docs/concepts/json_batching)."

(This lets readers know where they're going and helps them decide to follow the link or not.)

**Comment by mimrio** [comment]
Suggested changes applied. 
@lleonard-msft I sign-off from my part.

**Comment by lleonard-msft** [comment]
Hi @mimrio - Not sure why GH is reporting a merge conflict, but it is.  

Also, we need sign-off from @dkershaw10.

**Comment by mimrio** [comment]
Merge conflict solved.
@dkershaw10 can we get your sign-of here as well?
I have some urgency to move this API to V1 asap.

Thanks
Mario

**Comment by mimrio** [comment]
@lleonard-msft can I also use this PR to promote this documentation to GA (V1) or do I need to create another one? Thanks

**Comment by lleonard-msft** [comment]
@mimrio - Thanks for taking care of the conflict.  As soon as @dkershaw10 signs off, we should be good to go.  (Unless, of course,  something else comes up.)

Unfortunately, promotion to v1.0 is a separate PR.  Specific checklist guidelines are available in the internal on-boarding documentation.

**Comment by mimrio** [comment]
@dkershaw10 I really would like to close this asap. Can you please sign-off?

**Comment by lleonard-msft** [comment]
@mimrio - You'll want to review/resolve the merge conflicts.  If they're still there when Dan signs off, we're not going to be able to merge until they're dealt with.

**Comment by mimrio** [comment]
I'm not able to resolve the conflicts. I will close this PR and will create a new one.

**Comment by mimrio** [comment]
New PR here: https://github.com/microsoftgraph/microsoft-graph-docs/pull/3122

**Pull request closed without merging**